src/fileReader.ts

```diff
@@ -13,23 +13,43 @@ export async function* gameChunks(
   });
 
   let metadata: string[] = [];
+  let moves: string = '';
+  let ignoreGame: boolean = false;
   for await (const line of reader) {
-    // metadata & move lines chunked as a single game
-    if (line.startsWith('[')) {
-      metadata.push(line);
-    } else if (!line) {
-      // empty line, do nothing
+    if (line.startsWith('[Event')) {
+      // Start of a new game
+      ignoreGame = false;
```

Comment:
if we are ignoring game couldn't we just return? Then the generator will yield nothing. And it'll simplify this logic